man/systemd.service.xml

```diff
@@ -96,9 +96,12 @@
     <varname>After=</varname> on
     <filename>dbus.socket</filename>.</para>
 
-    <para>Socket activated service are automatically ordered after
+    <para>Socket activated services are automatically ordered after
     their activated <filename>.socket</filename> units via an
```

Comment:
hmm, I think we should at the same time change "activated → activating" here... After all the socket units are in the continous state of activating the service on events, they haven't finished it, hence -ing sounds more appropriate than past tense...

Of course, this is unrelated to your fix at hand, but maybe should be added to the same fix...